go/apps/simpleexample/SimpleClientCpp/helpers.cc

```diff
@@ -26,7 +26,7 @@
 #include <string>
 #include <thread>
 
-#include <taosupport.pb.h>
```

Comment:
Should we move this SimpleClientCpp to just cloudproxy/src/apps instead of where it currently is?